.github/workflows/pre_merge.yaml

```diff
@@ -84,6 +84,55 @@ jobs:
           curl -Os https://uploader.codecov.io/latest/linux/codecov
           chmod +x codecov
           ./codecov -t ${{ secrets.CODECOV_TOKEN }} --sha $COMMIT_ID -U $HTTP_PROXY -f .tox/coverage_unit-test-${{ matrix.tox-env }}.xml -F ${{ matrix.tox-env }}
+  Heavy-Unit-Test:
```

Comment:
@negvet please give your opinion on that stuff, can unit tests be so gpu-intensive, or is the builder that weak?

Reply:
From my perspective, we should try to avoid making unit tests to be intensive. But as I got Eugene's opinion, as for now, there are no options to fix it, so maybe we might revisit it later on when doing the whole test redesign.